packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx

```diff
@@ -48,13 +48,17 @@ export function useActiveVisits() {
       return null;
     }
 
-    let url = `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}&totalCount=true&location=${sessionLocation}`;
+    let url = `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}&`;
```

Comment:
Why are some properties converted into `urlSearchParams` and others left as is?

Reply:
I can't get the `customRepresentation` to get properly represented using `urlSearchParams.append()`. It looks like special characters in `customRepresentation` are getting encoded by default.

Reply:
I can't get the `customRepresentation` properly represented using `urlSearchParams.append()`. It looks like special characters in `customRepresentation` are getting encoded by default.